### PR TITLE
RabbitMQ Checker

### PR DESCRIPTION
**Pull request by Ph3nol**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,10 +22,12 @@
         "php": ">=5.3.3"
     },
     "require-dev": {
-        "sensiolabs/security-checker": "dev-master"
+        "sensiolabs/security-checker": "dev-master",
+        "videlalvaro/php-amqplib": "2.2.*"
```
Comment (lsmith77) [inline composer.json]: no need to add this here unless you also provide a test

**Comment by Ph3nol** [comment]
Following your advise, I just updated Composer dev requirements and squashed my commits!

**Comment by lsmith77** [comment]
needs a rebase

**Comment by Ph3nol** [comment]
Ready to be merged! :)

**Pull request merged**